nuttx-configs/px4fmu-v5/nsh/defconfig

```diff
@@ -750,7 +750,11 @@ CONFIG_MTD_BYTE_WRITE=y
 # CONFIG_MTD_N25QXXX is not set
 # CONFIG_MTD_SMART is not set
 CONFIG_MTD_RAMTRON=y
+# Ramtron writewait has a typo
+# in upstream NuttX - the CONFIG_
+# flag is missing.
 CONFIG_RAMTRON_WRITEWAIT=y
+RAMTRON_WRITEWAIT=y
```

Comment:
@davids5 This is broken currently. Can you put in a patch to NuttX and upstream this fix? See the ramtron driver for why I did that.